src/PostgREST/Query.hs

```diff
@@ -33,26 +34,37 @@ import Protolude hiding (Handler)
 
 -- The Queries that run on every request
 data MainQuery = MainQuery
-  { mqTxVars  :: SQL.Snippet       -- ^ the transaction variables that always run on each query
-  , mqPreReq  :: Maybe SQL.Snippet -- ^ the pre-request function that runs if enabled
+  { mqTxVars         :: SQL.Snippet       -- ^ the transaction variables that always run on each query
+  , mqPreReq         :: Maybe SQL.Snippet -- ^ the pre-request function that runs if enabled
   -- TODO only one of the following queries actually runs on each request, once OpenAPI is removed from core it will be easier to refactor this
-  , mqMain    :: SQL.Snippet
-  , mqOpenAPI :: (SQL.Snippet, SQL.Snippet, SQL.Snippet)
-  , mqExplain :: Maybe SQL.Snippet     -- ^ the explain query that gets generated for the "Prefer: count=estimated" case
+  , mqMain           :: SQL.Snippet
+  , mqOpenAPI        :: (SQL.Snippet, SQL.Snippet, SQL.Snippet)
+  , mqSetConstraints :: Maybe SQL.Snippet -- ^ the query that gets generated for the "Prefer: tx=rollback" case, https://www.postgresql.org/docs/current/sql-set-constraints.html
```

Comment:
I've ran `postgrest-coverage` locally, and it looks like CI is failing from this new field. The Logger addition is covered.